system/include/emscripten/emscripten.h

```diff
@@ -60,6 +60,13 @@ typedef void (*em_str_callback_func)(const char *);
 
 #define EMSCRIPTEN_KEEPALIVE __attribute__((used))
 
+#ifdef __wasm__
+#define EM_ENV_IMPORT(NAME) __attribute__((import_module("env"))) \
```

Comment:
Nit: can do `__attr__((a, b))` instead of `__attr__((a)) __attr((b))`

Reply:
Done.